netdata.spec.in

```diff
@@ -181,17 +181,18 @@ rm -rf "${RPM_BUILD_ROOT}"
 %defattr(-,root,root)
 
 %dir %{_sysconfdir}/%{name}
-
-%config(noreplace) %{_sysconfdir}/%{name}/*.conf
-%config(noreplace) %{_sysconfdir}/%{name}/charts.d/*.conf
-%config(noreplace) %{_sysconfdir}/%{name}/health.d/*.conf
-#%%config(noreplace) %{_sysconfdir}/%{name}/node.d/*.conf
-%config(noreplace) %{_sysconfdir}/%{name}/python.d/*.conf
-%config(noreplace) %{_sysconfdir}/%{name}/statsd.d/*.conf
+%dir %{_libdir}/%{name}
+
+%config(noreplace) %{_libdir}/%{name}/*.conf
+%config(noreplace) %{_libdir}/%{name}/charts.d/*.conf
+%config(noreplace) %{_libdir}/%{name}/health.d/*.conf
+#%%config(noreplace) %{_libdir}/%{name}/node.d/*.conf
+%config(noreplace) %{_libdir}/%{name}/python.d/*.conf
+%config(noreplace) %{_libdir}/%{name}/statsd.d/*.conf
```

Comment:
@paulfantom I think `noreplace` will do what it says, but we need it to replace these files.

Reply:
I don't know, maybe @alonbl can help?